src/SpineAnimationRenderer.ts

```diff
@@ -263,12 +294,10 @@ export class SpineAnimationRenderer extends Renderer {
    * @internal
    */
   override _onDestroy(): void {
-    const { _primitive, _subPrimitives } = this;
-    _subPrimitives.length = 0;
-    if (_primitive) {
-      _primitive.destroy();
-    }
     this._clearMaterialCache();
+    this._subPrimitives.length = 0;
+    this._primitive && this._primitive.destroy();
+    this._resource && this._resource.destroy();
```

Comment:
No! SpineResource is refer resource, You shouldn't do this

Reply:
done